packages/ui-components/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@mia-platform/ui-components",
+  "version": "0.1.0",
+  "license": "SEE LICENSE IN LICENSE",
+  "private": false,
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "source": "src/index.ts",
+  "typings": "dist/index.d.ts",
+  "dependencies": {
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-scripts": "4.0.3",
+    "web-vitals": "^1.0.1"
+  },
+  "peerDependencies": {
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1"
+  },
+  "scripts": {
+    "build": "rollup -c",
+    "lint": "eslint src --ext .ts",
+    "test": "craco test --runInBand",
```

Comment:
we should consider to remove craco in favour of react-scripts, as we don't use any custom configuration.